scripts/ci/push-image.sh

```diff
@@ -42,15 +42,15 @@
         beta_added=false
         stable_added=false
         for tag in "${TAGS_ARRAY[@]}"; do
-                echo "DEBUG: Processing tag: ${tag}"        
+                echo "DEBUG: Processing tag: ${tag}"
                 prerel="$(semver get prerel "${tag}" || true)"
                 version="$(semver get release "${tag}" || true)"
                 echo "  DEBUG: prerel: ${prerel}"
                 echo "  DEBUG: version: ${version}"
 
                 tags_array+=("okteto/okteto:${tag}")
                 echo "  DEBUG: Added tag to tags_array: okteto/okteto:${tag}"
-                echo "  DEBUG: tags_array so far: ${tags_array[@]}"
+                echo "  DEBUG: tags_array so far: ${tags_array[*]}"
```

Comment:
@jLopezbarb how can i check this is the same and working as expected? simulating the script¿

Reply:
Yes you can test it running: `scripts/ci/push-image.sh 1.23.1,1.2.3-beta.1asdasda,dev linux/amd64,linux/arm` and setting a exit just before the build

Reply:
ok, it works fine